packages/react/src/components/stepper/Stepper.stories.tsx

```diff
@@ -0,0 +1,307 @@
+      />
+
+      <div style={{ height: '300px', display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+        <Button
+          onClick={() => dispatch({ type: 'completeStep', payload: state.activeStep })}
+          style={{ height: 'fit-content', width: 'fit-content' }}
+        >
+          Complete step {state.activeStep}
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const Small = (args) => {
+  const reducer = commonReducer(5);
+
+  const initialState = {
+    activeStep: 1,
+    states: ['available', 'disabled', 'disabled', 'disabled', 'disabled'],
+  };
+  const labels = ['Step 1', 'Step 2', 'Step 3', 'Step 4 - longer text', 'Step 5'];
+
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  return (
+    <div style={{ maxWidth: '300px' }}>
+      <Stepper
+        labels={labels}
+        language="en"
+        small
+        states={state.states}
+        selectedStep={state.activeStep}
+        stepsTotal={5}
+        onStepClick={(event, number) => dispatch({ type: 'setActive', payload: number })}
+      />
+
+      <div style={{ height: '300px', display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+        <Button
+          onClick={() => dispatch({ type: 'completeStep', payload: state.activeStep })}
+          style={{ height: 'fit-content', width: 'fit-content' }}
+        >
+          Complete step {state.activeStep}
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const WithStepHeading = (args) => {
+  const reducer = commonReducer(5);
+
+  const initialState = {
+    activeStep: 1,
+    states: ['available', 'disabled', 'disabled', 'disabled', 'disabled'],
+  };
+  const labels = ['Step 1 - longer text', 'Step 2', 'Step 3', 'Step 4', 'Step 5'];
+
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  return (
+    <div>
+      <Stepper
+        labels={labels}
+        language="en"
+        stepHeading
+        states={state.states}
+        selectedStep={state.activeStep}
+        stepsTotal={5}
+        onStepClick={(event, number) => dispatch({ type: 'setActive', payload: number })}
+      />
+
+      <div style={{ height: '300px', display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+        <Button
+          onClick={() => dispatch({ type: 'completeStep', payload: state.activeStep })}
+          style={{ height: 'fit-content', width: 'fit-content' }}
+        >
+          Complete step {state.activeStep}
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const Overflow = (args) => {
```

Comment:
The animation is not used when the Stepper is scrolled with the ">" and "<" buttons. Is there a reason not to?

Reply:
When we use buttons '< >', the scrolling is done by a different javascript api endpoint, that does not have the animation options that the other api provided. And we can't really use the other api here, or atleast it would be way harder I think.

Reply:
It seems that with jQuery this would be possible: https://stackoverflow.com/a/6875094/4662347

If it's possible by jQuery, it should be possible by pure javascript too. I'll check on this.

Reply:
Was actually able to solve this by adding this css to the container:

`scroll-behavior: smooth`

But it does not seem to work on Safari. Actually none of the smooth animations of Stepper seem to work on Safari. So I guess it's a Safari 'feature' that we just have to live with.

Reply:
I noticed that the scroll options are not supported in Safari. But it jumps in the correct place, i.e works 👍

Reply:
Yeah true, for some reason the animations do not work on Safari =/